it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
+xSaveTheseValuesEncrypted = Ricorda questi dati. I dati archiviati non sono protetti da password.
+xSaveThisValueObscured = Ricorda questo valore. I valori archiviati non sono protetti da password.
+xSaveThisValueEncrypted = Ricorda questo valore. I valori archiviati non sono protetti da password.
 UserName = Nome Utente
 Password = Password
 SelectUserTitleLine = Seleziona Utente
-NoPasswordsEverSaved = Non sono mai state registrate password.
-Captured = I dati sono stati catturati.
-NotCaptured = Nulla da catturare.
-UnableToCapture = Impossibile catturare dati.
-Converted = I dati registrati sono stati convertiti.
-AllDataWillBeCleared = Tutti i dati registrati saranno eliminati.
-NotConverted = Impossibile convertire i dati registrati.
-PasswordNotChanged = Unable to change the password that protects your sensitive information.
-PasswordExpired = Your sensitive information is now locked.
-PasswordNotExpired = Unable to lock your sensitive information.
-Caveat = Saving Passwords and Other Sensitive Information##Password Manager and Form Manager will save passwords, user names, and other sensitive information and enter them for you automatically when they are required.#This sensitive information is stored on your computer in a file that's difficult, but not impossible, to read.#If other people have access to your computer, you may want to password protect the stored sensitive information by choosing a Master Password.#If you choose to password protect your stored information, you will be asked to provide your Master Password from time to time.  This approach provides better security but is slightly less convenient.
+NoPasswordsEverSaved = Non sono mai state salvate password.
+Captured = I dati sono stati acquisiti.
+NotCaptured = Nulla da acquisire.
+UnableToCapture = Impossibile acquisire dati.
+Converted = I dati archiviati sono stati convertiti.
+AllDataWillBeCleared = Tutti i dati archiviati saranno eliminati.
+NotConverted = Impossibile convertire i dati archiviati.
+PasswordNotChanged = Impossibile cambiare la password che protegge i tuoi dati sensibili.
+PasswordExpired = I tuoi dati sensibili sono stati messi in sicurezza.
+PasswordNotExpired = Impossibile mettere in sicurezza i tuoi dati sensibili.
+Caveat = Salvare password e altri dati sensibili## Il Gestore password e il Gestore moduli salveranno le password, i nomi utenti e altri dati sensibili e li inseriranno per te automaticamente quando sono richiesti.#Questi dati riservati sono archiviati sul tuo computer in un file che è difficile, ma non impossibile, leggere.#Se altri utenti hanno accesso al tuo computer, puoi proteggere con password i dati sensibili archiviati scegliendo una Password principale.#Se scegli di proteggere con password i tuoi dati riservati, ti sarà richiesto di fornire di volta in volta la tua Password principale. Questo metodo fornisce una miglior sicurezza ma è leggermente più scomodo.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

Salvare password e altri dati sensibili##Il Gestore password e il Gestore moduli salveranno le password, i nomi utente e altri dati sensibili e li inseriranno automaticamente per te quando necessario.#Questi dati riservati sono archiviati sul tuo computer in un file che è difficile ma non impossibile da leggere.#Se altri utenti hanno accesso al tuo computer, puoi proteggere con password i dati sensibili archiviati scegliendo una Password principale.#Se scegli di proteggere con password i tuoi dati riservati, ti sarà richiesto di fornire di volta in volta la tua Password principale. Questo metodo fornisce una miglior sicurezza ma è leggermente più scomodo.